### PR TITLE
Added context menu to TV episode list items

### PR DESCRIPTION
**Pull request by poisdeux**
The context menu adds play and queue functionality for tv show
episode items in the tv show episode list.

Replaced checkmark icon with the context menu icon. Now watched episodes are visualised by changing the colour of the episode number, title, and context menu icon.

This implements feature request #187

### CODE REVIEW

File: app/src/main/java/org/xbmc/kore/ui/TVShowEpisodeListFragment.java [hunk 1]
```diff
@@ -549,11 +552,20 @@ public void bindChildView(View view, Context context, Cursor cursor, boolean isL
...
-                viewHolder.checkmarkView.setColorFilter(themeAccentColor);
+                viewHolder.episodenumberView.setTextColor(themeAccentColor);
+                viewHolder.titleView.setTextColor(themeAccentColor);
+                viewHolder.contextMenuView.setColorFilter(themeAccentColor);
             } else {
-                viewHolder.checkmarkView.setVisibility(View.GONE);
+                viewHolder.episodenumberView.setTextColor(defaultTextColor);
+                viewHolder.titleView.setTextColor(defaultTextColor);
+                viewHolder.contextMenuView.setColorFilter(themeAccentColor);
+                viewHolder.contextMenuView.clearColorFilter();
```
Comment (SyncedSynapse) [inline app/src/main/java/org/xbmc/kore/ui/TVShowEpisodeListFragment.java]: Any reason for setting the color filter and then clearing it?

**Comment by SyncedSynapse** [comment]
I'm not sure about coloring the episode number/title/menu with the accent color for watched episodes. That tends to draw attention to those episodes, which is exactly the opposite that should be done.

How about using the secondary color `?attr/appTextColorSecondary`.

File: app/src/main/java/org/xbmc/kore/ui/TVShowEpisodeListFragment.java (hunk 1 above)
Comment (poisdeux) [inline app/src/main/java/org/xbmc/kore/ui/TVShowEpisodeListFragment.java]: Nope, copy/paste leftover, which I forgot to remove.

**Comment by poisdeux** [comment]
I see what you mean. However, instead of changing the text colour I think adding the checkmark before the episode number is a better solution. I don't have any time right now but I'll do it ASAP.

**Comment by poisdeux** [comment]
Changed the layout and added the checkmark again. Hope you like it.

**Comment by SyncedSynapse** [comment]
Looks good.

**Pull request merged**